### ISSUE TITLE
Option to add parentheses to multi-line continuations

### ISSUE THREAD

**Issue by ranocha** [domluna/JuliaFormatter.jl#709]
Is there an option to add parentheses when expressions involving binary operators are continued implicitly, i.e., to turn

```julia
a = the_first_very_long_expression *
    the_second_very_long_expression
```

into something like

```julia
a = (the_first_very_long_expression *
     the_second_very_long_expression)
```

**Comment by domluna** [comment]
not currently. what would this look like? if the line is broken add parenthesis?

**Comment by sloede** [comment]
> if the line is broken add parenthesis?

Yes, exactly. If a line is turned into multiple line due to line length restrictions, wrap it in parentheses (if not already inside parentheses, eg, inside a function argument tuple).
